sympy/functions/elementary/complexes.py

```diff
@@ -654,6 +710,10 @@ class arg(Function):
     pi/2
     >>> arg(sqrt(2) + I*sqrt(2))
     pi/4
+    >>> arg(sqrt(3)/2 + I/2)
+    pi/6
+    >>> arg(4/5 + 3*I/5)
+    0.643501108793284
```

Comment:
I think this should either use Rational like `(4 + 3*I)/5` or use floats explicitly like `0.8 + 0.6*I`.

Reply:
Done.

Reply:
I think this is still using a float.